src/index.d.ts

```diff
@@ -0,0 +1,5 @@
+type Locale = "ar" | "de" | "en" | "es" | "fr" | "he" | "it" | "ja" | "ko" | "nl" | "pt-BR" | "ru" | "zh-Hans" | "zh-Hant"
```

Comment:
I'm not familiar with typescript definition, is there a way to use/import the values defined [here](https://github.com/Locale/amplify-i18n/blob/main/src/index.js#L4-L19)?
I'm pretty sure at a time the lists won't be up to date.

Reply:
I will take a look at this, I'm pretty certain it's possible :)

Reply:
Without using proper typescript it's sadly not possible using a .js-defined array as type. Neither I am able to import the array from a JSON-File and import this as a type. Thus, it's easier to convert the whole logic to typescript or maintain languages manually to the definition file.